src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/OGRSAssessmentRepository.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.justice.digital.delius.jpa.standard.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import uk.gov.justice.digital.delius.jpa.standard.entity.OGRSAssessment;
+
+@Repository
+public interface OGRSAssessmentRepository extends JpaRepository<OGRSAssessment, Long> {
+
+    @Query("select assessment from OGRSAssessment assessment inner join assessment.event event where event.offenderId = :offenderId and assessment.softDeleted = 0 and assessment.lastUpdatedDate = \n" +
+        "    (select max(assessmentLatest.lastUpdatedDate) \n" +
+        "    from OGRSAssessment assessmentLatest \n" +
+        "    inner join assessmentLatest.event eventLatest \n" +
+        "    where eventLatest.offenderId = :offenderId \n" +
+        "    and assessmentLatest.softDeleted = 0)")
+    OGRSAssessment findByOffenderId(@Param("offenderId") Long offenderId);
```

Comment:
maybe rename this as findLatestByOffenderId

Reply:
Good idea, done

Reply:
Good idea, done